app/src/main/res/values/strings.xml

```diff
@@ -106,6 +106,7 @@
     <string name="light">Material Light</string>
     <string name="dark">Material Dark</string>
     <string name="daytime">Daytime</string>
+    <string name="black">Black</string>
```

Comment:
It would be better if you explain that it would reduce the battery consumption in AMOLED devices.

Reply:
If you think that'd be fitting but it's a bit wordy. 
It's just a black theme on non-OLED-based screens. I don't know if it's worth explaining as if it's a feature of the theme when that's device dependent.

Reply:
If you can't add a smaller explanatory text (like "saves battery on AMOLED") "Black (for AMOLED)" is good enough.

Reply:
Pitch black does save battery for AMOLED devices... It's a universal truth... 
As for PR, I'll review later.
Thanks for the contribution @PiwwowPants